stubs/third_party_taint/boto3_clients_sources_sinks.pysa

```diff
@@ -0,0 +1,16 @@
+# Requires mypy_boto3 stubs and explicit type annotations to work because the stubs
+# for boto3.get_client("client-name") defines it to return `typing.any`.
+
+def mypy_boto3_acm.client.ACMClient.export_certificate() -> TaintSource[ServerSecrets]: ...
+
+def mypy_boto3_amplifybackend.client.AmplifyBackendClient.create_token() -> TaintSource[ServerSecrets]: ...
+def mypy_boto3_amplifybackend.client.AmplifyBackendClient.get_backend_auth() -> TaintSource[ServerSecrets]: ...
+def mypy_boto3_amplifybackend.client.AmplifyBackendClient.get_token() -> TaintSource[ServerSecrets]: ...
+
+def mypy_boto3_apigateway.client.APIGatewayClient.create_api_key() -> TaintSource[ServerSecrets]: ...
+def mypy_boto3_apigateway.client.APIGatewayClient.get_api_key() -> TaintSource[ServerSecrets]: ...
+def mypy_boto3_apigateway.client.APIGatewayClient.get_api_keys() -> TaintSource[ServerSecrets]: ...
+def mypy_boto3_apigateway.client.APIGatewayClient.get_authorizer() -> TaintSource[ServerSecrets]: ...
+def mypy_boto3_apigateway.client.APIGatewayClient.get_client_certificate() -> TaintSource[UserSecrets]: ...
```

Comment:
We usually use `UserSecrets` to indicate secrets belonging to a user that is interacting with a web server. In this case, a client certificate is for one server to interact with another. So we should probably label these two as ServerSecrets as well.

Reply:
@gbleaney oh gotcha, done :)